book/P1C5_eha.qmd

```diff
@@ -12,159 +12,456 @@ abstract: TODO (150-200 WORDS)
+  & =  \Prodi_{u \in (\zeta,\tau]}(\mathbf{I} + d\mathbf{H}(u)),\nonumber\\
+\end{aligned}
+$$ {#eq-ms-prodint-cumu-hazards}
+<!--  -->
+where $\mathbf{I}$ is a $(q+1) \times (q+1)$ identity matrix and $d\mathbf{H}(u)$ is the matrix of increments of the cumulative hazard within infinitesimally small intervals.
+Thus, similar to the competing risks setting, relationship @eq-ms-prodint implies that knowledge of the transition specific (cumulative) hazards is sufficient to fully specify the multi-state process.
+
+As analytical solutions of @eq-ms-prodint only exist for specific models, in practice, the transition probabilities are often once again approximated numerically via a finite matrix product on a discrete time grid $\zeta = t_0 < t_1 < \cdots < t_{J-1} < t_{J} = \tau$
+<!--  -->
 $$
+\mathbf{P}(\zeta, \tau) \approx \prod_{j=1}^{J}(\mathbf{I} + \Delta\mathbf{H}_{\ell e}(t_{j})),
+$$ {#eq-ms-prodint-approx}
+<!--  -->
+where $\triangle H_{\ell e}(t_{j}) = H_{\ell e}(t_{j}) - H_{\ell e}(t_{j-1})$ is the increment of the cumulative hazards between to consecutive time points.
 
-$\bullet$ indicates impossible transitions, as in the illness-death model there are no transitions from higher to lower states.
 
+### Aalen-Johanson Estimator for Transition Probabilities {#sec-ms-aalen-johanson}
 
+Per @eq-ms-prodint-approx all we need for estimation of transition probabilities is an estimate of the transition specific cumulative hazards $H_{\ell e}(\tau)$.
+Luckily, similar consideration as for competing risks in @sec-aalen-johanson apply, replacing the cause-specific hazards (@eq-cs-discrete-hazard) with transition-specific hazards
+<!--  -->
+$$
+h^{d}_{\ell e}(t_{(k)}):= \frac{d_{\ell e,t_{(k)}}}{n_{\ell; t_{(k)}}},
+$$
+<!--  -->
+where
 
-## Recurrent Events {#sec-recurrent-events}
+- $d_{\ell e,t_{(k)}}$ is now the number of observations with transition $\ell \rightarrow e$ at time $t_{(k)}$ and
+- $n_{\ell; t_{(k)}} = |\mathcal{R}_{\ell;t_{(k)}}|$ is the number of subjects in state $\ell$ at time $t_{(k)}$.
 
-The *recurrent events* setting is defined by the same event type occurring multiple times.
-While the recurrent events setting can be viewed as a special case of the multi-state setting (@sec-multi-state), we treat it here separately as it can also be approached as single event data with correlation.
-Some examples of recurrent events include recurrence of infectious diseases, episodes of epilepsy, and replacements of machine parts.
+Thus, risk set $\mathcal{R}_{\ell;t_{(k)}}$ excludes subjects who entered a terminal state or were censored before time $t_{(k)}$, but also subjects who are not in state $\ell$ at that time (either because they did not reach it yet or transition to another state occurred beforehand).
+
+The cumulative transition-specific hazards thereby follow as
+<!--  -->
+$$
+H_{NA,\ell e}(\tau) = \sum_{k:t_{(k)}\leq \tau} h^d_{\ell e}(t_{(k)}) = \sum_{k:t_{(k)}\leq \tau}\frac{d_{\ell e,t_{(k)}}}{n_{\ell; t_{(k)}}},
```

Comment:
Arguably you don't need the far RHS because it's just been stated in the equation above and by now readers will be very used to seeing this format (i.e., H = sum h = sum d/n)